tsl/src/nodes/decompress_chunk/decompress_chunk.c

```diff
@@ -139,6 +139,8 @@ append_ec_for_seqnum(PlannerInfo *root, CompressionInfo *info, SortInfo *sort_in
 	newec->ec_max_security = 0;
 	newec->ec_merged = NULL;
 
+	info->compressed_rel->eclass_indexes =
```

Comment:
Is this a separate optimization from the one in `create_compressed_scan_paths`?

Reply:
Its more of a fixed oversight that the optimization depends on. Never caused problems because we never join on sequence number column.